src/packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -915,45 +915,42 @@ new PrismaClient({
      * @param middlewareIndex
      * @returns
      */
-    private _request(
+    private async _request(
```

Comment:
```suggestion
    private _request(
```

This has performance implications, it's not needed.